### PR TITLE
[code] update stable to 1.64

### PR DESCRIPTION
**Pull request by jeanp413**
## Description

Updates stable VS Code Web to 1.64
VSCode Endgame: https://github.com/microsoft/vscode/issues/141298

## Related Issue(s)

Fixes #

## How to test

- Select vscode stable in dashboard settings
- Start a workspace.
- Use about dialog to check that version is 1.64.

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **jeanp413** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**
- **[installer/pkg/components/workspace/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/pkg/components/workspace/ide/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by filiptronicek — APPROVED** [review]
LGTM ✅ :shipit: 

image

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 062b662966a4be42a496d5ea29e000edd4d3c6db</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/8019#pullrequestreview-872286634" title="Approved">filiptronicek</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [filiptronicek]
- ~~[installer/pkg/components/workspace/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/pkg/components/workspace/ide/OWNERS)~~ [filiptronicek]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/8019#pullrequestreview-872286634" title="Approved">filiptronicek</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/8019#issuecomment-1029393345" title="Approved">jeanp413</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/8019#issuecomment-1029393345" title="Approved">jeanp413</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [filiptronicek,jeanp413]
- ~~[installer/pkg/components/workspace/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/pkg/components/workspace/ide/OWNERS)~~ [filiptronicek,jeanp413]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
